pymodels/SI_V25_04/fitted_models.py

```diff
@@ -0,0 +1,140 @@
+"""Contain magnets strengths that fit real machine with storage ring model.
+
+Different models may reproduce different aspects of the real machine.
+
+"""
+
+import numpy as _np
+
+import pyaccel as _pa
+
+from .families import get_family_data
+
+
+def vertical_dispersion_and_coupling(model, famdata=None):
+    """Reproduce vertical dispersion function and betatron coupling of machine.
+
+    This model was built via fitting of the measured vertical dispersion
+    function with the chromatic skew quadrupoles of the machine. The
+    measurement used for fitting was the matrix measurement of folder:
+        shared/screens-iocs/data_by_day/2022-05-24-SI_LOCO/
+    with name:
+        respmat_endofmay22_bpms_03sector_switching_issue.pickle
+
+    The transverse tunes were also adjusted.
+
+    The version of the storage ring model used in this fitting was:
+        SI_V25_04
+    The Head of the repository was in commit:
+        b806c3bcca6a8a8e02c1a3c9abee92f7366c44cb
+
+    Example:
+        >> from pymodels import si
+        >> import pyaccel as pa
+        >> modcorr = si.create_accelerator()
+        >> modcorr = si.fitted_models.vertical_dispersion_and_coupling(modcorr)
+        >> eqpar = pa.optics.EqParamsFromBeamEnvelope(modcorr)
+        >> print(eq_fit.emit1*1e12, eq_fit.emit2*1e12)
+        (247.85215848346252, 4.728953607736642)
+        >>print(eq_fit.tune1, eq_fit.tune2, eq_fit.tune3)
+        (0.07774444288752377, 0.14136042194137038, 0.004712773241401889)
+
+    Args:
+        model (pyaccel.accelerator.Accelerator): Sirius storage ring model.
+        famdata (dict, optional): Family data. Defaults to None.
+
+    Returns:
+        (pyaccel.accelerator.Accelerator): Fitted Storage ring model.
+
+    """
+    if famdata is None:
+        famdata = get_family_data(model)
+    idcs_qn = _np.array(famdata['QN']['index']).ravel()
+    idcs_qs = _np.array(famdata['QS']['index']).ravel()
+
+    stren_qn = _np.array([
+        +0.71430178, -0.22654698, +0.56367412, +0.86806588, +0.64368619,
```

Comment:
@fernandohds564 , I think you could add to this python module the function(s) you used for fitting the quads. and maybe another argument to vertical_dispersion_and_coupling` in case the user prefer running the fitting anew for the given model, using the added function(s)

Reply:
I think @murilobalves  will add the method you required here in some part of your apsuite package. I don't think here is the right place for it, since it may depend on other packages, such as apsuite.

Reply:
I created a PR in apsuite with this script: https://github.com/lnls-fac/apsuite/pull/188

Reply:
ok.